apps/SageAccountWeb/src/LoginPage.tsx

```diff
@@ -26,8 +27,20 @@ function LoginPage(props: LoginPageProps) {
   return (
     <StyledOuterContainer>
       <StyledInnerContainer>
-        <Box sx={{ py: 10, px: 8, height: '100%', position: 'relative' }}>
-          <Box sx={{ minHeight: '530px' }}>
+        <Box
+          sx={{
+            py: 10,
+            px: 8,
+            height: '100%',
+            position: 'relative',
+            '.login-flow-back-button': backButtonSx,
```

Comment:
Style the back button provided by StandaloneLoginForm, included in this location to show which component the back button's absolute style is relative to